website/content/about/welcome.md

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
```suggestion
---
title: Welcome to the CNCF Environmental Sustainability Technical Advisory Group (TAG)
description: The CNCF Environmental Sustainability Technical Advisory Group (TAG) is a community of experts focused on promoting sustainable practices and reducing the environmental impact of cloud native technologies.
---

```
We are missing the header here (feel free to change the title and description) :)

Reply:
Yes, I agree on the duplication. I was playing around with. Let me fix.